conversions/hexadecimal_to_decimal

```diff
@@ -0,0 +1,48 @@
+def hexToDecimal(hex_string: str) -> int:
+    """
+    converts a given hexadecimal value to its decimal value
+    >>> hexToDecimal("a")
+    10
+    >>> hexToDecimal("")
+    ValueError: Empty string value was passed to the function
+    >>> hexToDecimal("-12m")
+    ValueError: None hexadecimal value was passed to the function
+    >>> hexToDecimal("12f")
+    303
+    >>> hexToDecimal("   12f   ")
+    303
+    """
+    decimal_number = 0
+    hex_table = {
+        "1": 1,
+        "2": 2,
+        "3": 3,
+        "4": 4,
+        "5": 5,
+        "6": 6,
+        "7": 7,
+        "8": 8,
+        "9": 9,
+        "A": 10,
+        "B": 11,
+        "C": 12,
+        "D": 13,
+        "E": 14,
+        "F": 15,
+    }
```

Comment:
Convert hex_table to a global variable

Reply:
hex_lookup = {hex(i)[2:].upper(): i for i in range(16)}  # for uppercase
hex_lookup = {hex(i)[2:]: i for i in range(16)}  # for lowercase (preferred)